src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,42 @@
+.watch {
+  @include clock;
+  background-color: black;
+  width: $watch-size;
+  height: $watch-size;
+  margin-top: 50px;
+  border: 4px solid $main-color;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+
+  &__center {
+    @include design;
+    @include clock;
+    height: $center-size;
+    width: $center-size;
+  }
+
+  &__seconds {
+    @include design;
+    width: 1%;
+    height: 36%;
+    top: 14%;
```

Comment:
it's not clear what these value means
will be better to calculate them somehow depending on the size of watch / hands